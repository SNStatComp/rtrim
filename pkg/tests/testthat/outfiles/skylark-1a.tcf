FILE outfiles/skylark.dat
TITLE  skylark-1a
NTIMES 8
NCOVARS 2
LABELS
Habitat
Cov2
End
MISSING 999
WEIGHT Absent

comment Example 1; without overdispersion or autocorrelation"
weighting off
overdisp off
serialcor off
model 3
