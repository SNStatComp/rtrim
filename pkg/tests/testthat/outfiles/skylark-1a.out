 TRIM 3.61 :  TRend analysis and Indices for Monitoring data 
 STATISTICS NETHERLANDS 

 Date/Time: 4-7-2016 15:06:11

 Title :   skylark-1a

 Comment: Example 1; without overdispersion or autocorrelation"

 The following  5 variables have been read from file: 
 F:\TRIM\TRIM_manual_demo\skylark.dat

 1. Site             number of values:    55
 2. Time             number of values:     8
 3. Count            missing =        999
 4. HABITAT          number of values:  2
 5. COV2             number of values:  4

 Number of sites without positive counts (removed)       0

 Number of observed zero counts           0
 Number of observed positive counts     202
 Total number of observed counts        202
 Number of missing counts               238
 Total number of counts                 440

 Total count                         2536.0

 Sites containing more than 10% of the total count 
   Site Number  Observed Total    %   
         3        431.0         17.0
        37        266.0         10.5
        40        624.0         24.6

 Time Point Averages 
   TimePoint  Observations  Average   Index  
       1              25       8.52    1.00
       2              20       8.10    0.95
       3              30      10.90    1.28
       4              30      11.27    1.32
       5              28      12.75    1.50
       6              29      14.66    1.72
       7              22      17.00    2.00
       8              18      18.89    2.22

 RESULTS FOR MODEL: Effects for Each Time Point
 ----------------------------------------------


 ESTIMATION METHOD = Maximum Likelihood


  Total time used:  5.08 seconds

 GOODNESS OF FIT
  Chi-square                188.14, df    140, p 0.0041
  Likelihood Ratio          184.98, df    140, p 0.0065
  AIC (up to a constant)    -95.02

 WALD-TEST FOR SIGNIFICANCE OF DEVIATIONS FROM LINEAR TREND 
  Wald-Test             20.29, df     6, p  0.0025

 PARAMETER ESTIMATES 

 Parameters for Each Time Point

  Time    Additive    std.err.   Multiplicative  std.err.
     1      0                          1
     2     -0.3430      0.1086         0.7096      0.0771
     3     -0.1732      0.0927         0.8410      0.0780
     4     -0.1875      0.0932         0.8290      0.0773
     5     -0.0853      0.0915         0.9182      0.0840
     6      0.0213      0.0902         1.0216      0.0922
     7      0.0953      0.0924         1.1000      0.1016
     8      0.1712      0.0940         1.1867      0.1115

 Linear Trend + Deviations for Each Time 

           Additive    std.err.   Multiplicative  std.err.
 Slope      0.0485      0.0107         1.0497      0.0112

 Time Deviations
    1       0.2325      0.0539         1.2617      0.0680
    2      -0.1591      0.0670         0.8529      0.0571
    3      -0.0378      0.0541         0.9629      0.0521
    4      -0.1006      0.0539         0.9043      0.0488
    5      -0.0469      0.0505         0.9542      0.0481
    6       0.0112      0.0466         1.0113      0.0472
    7       0.0366      0.0453         1.0373      0.0470
    8       0.0640      0.0437         1.0661      0.0466

 Time INDICES
  Time     Model     std.err.     Imputed    std.err.
     1     1                       1
     2     0.7096     0.0771       0.7096     0.0771
     3     0.8410     0.0780       0.8410     0.0780
     4     0.8290     0.0773       0.8290     0.0773
     5     0.9182     0.0840       0.9182     0.0840
     6     1.0216     0.0922       1.0216     0.0922
     7     1.1000     0.1016       1.1000     0.1016
     8     1.1867     0.1115       1.1867     0.1115

 TIME TOTALS
  Time      Model    std.err.     Imputed    std.err.
     1        511         38          511         38
     2        362         31          362         31
     3        429         26          429         26
     4        423         25          423         25
     5        469         27          469         27
     6        522         27          522         27
     7        562         32          562         32
     8        606         36          606         36

 OVERALL SLOPE MODEL: Moderate increase (p<0.01) **
  Additive      std.err.   Multiplicative  std.err.
    0.0485       0.0107         1.0497      0.0112

 OVERALL SLOPE IMPUTED+(recommended): Moderate increase (p<0.01) **
  Additive      std.err.   Multiplicative  std.err.
    0.0485       0.0107         1.0497      0.0112

